### PR TITLE
Create new command to install latest version

### PR DESCRIPTION
**Pull request by sanchitgera**
## Description
Give users the option to install the latest version of Yarn. Using `yvm install --latest` or `yvm install -l`. 
See #175 

### DevQA Steps
- Run `yvm ls-remote`. Note the latest version of Yarn available 
- Run `yvm install --latest`
- See that the version is installed 
- Run `yvm install --latest` again 
- See that yvm prints a message about version already installed

### CODE REVIEW

**Comment by jakebolam** [comment]
@sanchitgera you should put up a PR to add yourself with an app-code badge now too!
https://github.com/tophat/yvm#contributors

File: src/yvm.js [hunk 1]
```diff
@@ -33,11 +33,16 @@ if (!process.argv.includes('exec')) {
...
+    .option('-l, --latest', 'Install the latest version of Yarn available')
     .description('Install the specified version of Yarn.')
-    .action((maybeVersion) => {
+    .action((maybeVersion, command) => {
         const [version] = getSplitVersionAndArgs(maybeVersion);
-        const install = require('./commands/install');
-        install(version);
+        const {installVersion, installLatestVersion} = require('./commands/install');
+
+        if(command.latest) {
```
Comment (jakebolam) [inline src/yvm.js]: This guard should be before line 39
`const [version] = getSplitVersionAndArgs(maybeVersion);`

File: .all-contributorsrc [hunk 2]
```diff
@@ -89,7 +89,8 @@
       "avatar_url": "https://avatars0.githubusercontent.com/u/8632167?v=4",
       "profile": "http://www.sanchitgera.ca",
       "contributions": [
```
Comment (jakebolam) [inline .all-contributorsrc]: You need to run `yarn contributors:generate` to update the readme

Comment (sanchitgera) [reply .all-contributorsrc]: I'll put this in a follow up PR. Shouldn't combine it with a feature branch.

**Comment by sanchitgera** [comment]
👍 Holding off on the merge till QA

**Comment by jakebolam** [comment]
QA LGTM

**Comment by sanchitgera** [comment]
@jakebolam Could you approve again?

**Pull request merged**